ui/components/app/flask/experimental-area/index.scss

```
@@ -0,0 +1,44 @@
+.experimental-area {
+  color: $flask-purple;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .logo {
+    padding: 20px;
+    line-height: 0.625rem;
+    font-family: monospace;
+  }
+
+  .experimental-text {
+    padding: 20px;
+    font-family: monospace;
+    margin: auto;
+    line-height: 16px;
+  }
+
+  .text {
+    padding: 20px;
+    max-width: 670px;
+  }
+
+  ul {
+    padding: 20px 0;
+  }
+
+  li {
+    list-style-type: disc;
+    padding-left: 10px;
+    list-style-position: inside;
+  }
+
+  button {
+    background-color: $flask-purple !important;
+    border: 0 !important;
+    color: #f2f3f4;
```

Use the design system variable for this color. We should refrain from using static #hex values
```suggestion
    color: $ui-1;
```